public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
can you please remove `public/mainfest.json`, `public/logo192.png`, `public/logo512.png` files as you don't need them?

Reply:
Ok, I have removed them.